### ISSUE TITLE
Prepare eecology dataset 

### ISSUE THREAD

**Issue by cwmeijer** [NLeSC/mcfly#49]
Gull dataset

**Comment by dafnevk** [comment]
I managed to get NaN again!
When I train the best model (an DeepConvLSTM model) with 50 epochs, it already finds it best validation accuracy after 6 epochs. after 38 epochs the loss becomes NaN.
(I didn't fix the seed, so I don't know if this error is reproducible).
I also got this for one other DeepConvLSTM model.

We haven't implemented batch norm in DeepConvLSTM so that might be a reason. We could easily implement it in the Conv layers of the DeepConvLSTM model.
